tin/static/css/base.css

```diff
@@ -141,7 +141,7 @@ body {
     text-align: right;
 }
 
-ul:not(.standard) {
+ul.no-list {
```

Comment:
With this change, you should also remove the `standard` class from any `<ul>`'s that have it.